Sources/Fluent/Relations/Children.swift

```diff
@@ -13,10 +13,8 @@ extension Children: QueryRepresentable {
         guard let ident = parent.id else {
             throw RelationError.noIdentifier
         }
-
-        let query = try T.query()
-
-        let foreignId = foreignKey ?? "\(type(of: parent).name)_\(query.idKey)"
+        
+        let foreignId = foreignKey ?? "\(type(of: parent).name)_\(T.idKey)"
```

Comment:
While we're in here, this reads a little weird to me now that we've clarified the latter.

`"\(T.name)_\(T.idKey)"` would be more understandable I think.